### ISSUE TITLE
Temperature adjustement does not applies on all monitors (Multi GPU)

### ISSUE THREAD

**Issue by dethegeek** [jonls/redshift#422]
Hi

I started using RedShift a few days ago.

It ran sucessfully with a Nvidia graphic card, 3 monitors, and drived by Nouveau.

I added a identical 2nd graphic card to use 4 monitors (2 on each GPU). This setup, more complex, forced me to move to the proprietary drivers from NVidia (I hope this is not definitive). With this later setup, Redshift fails to adjust the temperature on 3 monitors.

I'm expecting my xorg.conf may be useful. If so, pease, tell me and I'll post it here.

**Comment by maandree** [comment]
I don't think the xorg.conf file is too interesting, it could however be interesting to know if you have single-screen or multi-screen setup. Are all monitors reported under "Screen 0" when you run `xrandr`?

**Comment by dethegeek** [comment]
Hi

Now I switched to Nvidia proprietary driver, xrandr does not work anymore 
```
localhost:~$ xrandr
RandR extension missing
```
I just found nvidia drivers don't support xrandr since xrandr 1.1, so it appears to be not a bug or config issue.

I got 1 screen per monitor, then 4 screens as I defined in xorg.conf, and reported by the nvidia configuration GUI.

![image](https://cloud.githubusercontent.com/assets/14139801/22149770/f6225a0a-df15-11e6-85ce-daac49f1b014.png)

Note there are 8 entries in  the dropdown. Selecting a screen is equivalent to selecting one of the monitors,
